README.md

```diff
@@ -20,7 +26,7 @@ const keyborg = createKeyborg(window);
 // This is called every time the keyboard input state changes
 const handler = (isUsingKeyboard) => {
   if (isUsingKeyboard) {
-    document.addAttribute('data-is-keyboard', 'true');
```

Comment:
- `addAttribute` is not existing method, use `setAttribute` 
- `setAttribute` cannot be called on `document`